sklearn/dummy.py

```diff
@@ -51,7 +52,7 @@ class DummyClassifier(MultiOutputMixin, ClassifierMixin, BaseEstimator):
              The default value of `strategy` has changed to "prior" in version
              0.24.
 
-    random_state : int, RandomState instance or None, optional, default=None
+    random_state : int or RandomState instance or None, default=None
```

Comment:
```suggestion
    random_state : int or RandomState instance, default=None
```

Reply:
I think None should be explicited for random_state: https://github.com/scikit-learn/scikit-learn/pull/17306#discussion_r432573500

Reply:
True. We have inconsistence here.
One PR to fix them all would be great.